content/working-groups/server/extended-validation/documents/index.md

```diff
@@ -11,10 +11,12 @@ title: EV Guidelines for TLS Server Certificates
 
 ### Current Version
 
-[EV TLS Server Certificate Guidelines 1.8.0][1] – ([redlined][2]) – adopted by Ballot [SC56][3]
+[EV TLS Server Certificate Guidelines 1.8.1][181] – ([redlined][181r]) – adopted by Ballot [SC68][sc68]
```

Comment:
Alternatively you can write:
```suggestion
[EV TLS Server Certificate Guidelines 1.8.1](EVG-1.8.1.pdf) – ([redlined](EVG-1.8.1-redlined.pdf)) – adopted by Ballot [SC68][sc68]
```
If you adopt you will need to delete the links at the end of the document.

Reply:
I´d prefer to leave it with the links at the end for clarity. At least at the moment until we decide, if possible, an harmonized way to do it.